ie-comunicaciones.csl

```diff
@@ -0,0 +1,205 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" and="text" delimiter-precedes-et-al="never" delimiter-precedes-last="never" et-al-min="2" et-al-use-first="1" et-al-subsequent-min="2" et-al-subsequent-use-first="1" demote-non-dropping-particle="sort-only" default-locale="en-US">
```

Comment:
given that you can publish in Spanish and Portuguese in the journal, we'll want the default-locale removed.
Also, please remove the et-al settings here, better to set them only once